.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+// These are GitHub Codespaces specific settings.
+{
+  "name": "Wasp Todo App Tutorial",
+  "customizations": {
+    "vscode": {
+      "extensions": ["wasp-lang.wasp", "PKief.material-icon-theme"],
+      "settings": {
+        "workbench.iconTheme": "material-icon-theme"
+      }
+    }
+  },
+  "containerEnv": {
+    "WASP_TELEMETRY_CONTEXT": "codespaces"
+  },
+  "forwardPorts": [3000, 3001],
+  "portsAttributes": {
+    "3000": {
+      "label": "React-Vite Client"
+    },
+    "3001": {
+      "label": "NodeJS Server"
+    }
+  },
+  "postCreateCommand": "curl -sSL https://get.wasp-lang.dev/installer.sh | sh -s",
+  "postStartCommand": "printf 'export WASP_WEB_CLIENT_URL=https://$CODESPACE_NAME-3000.app.github.dev\n' >> $HOME/.bashrc && printf 'export REACT_APP_API_URL=https://$CODESPACE_NAME-3001.app.github.dev\n' >> $HOME/.bashrc && echo 'gh codespace ports visibility 3000:public -c $CODESPACE_NAME\n' >> $HOME/.bashrc && echo 'gh codespace ports visibility 3001:public -c $CODESPACE_NAME\n' >> $HOME/.bashrc",
+  "workspaceFolder": "/workspaces/wasp/examples/tutorials/TodoApp"
```

Comment:
How do you determine this folder, what does it mean?

Reply:
This means that the codespace editor will open up the directory at that path, which is the tutorial todo app, rather than the entire main wasp repo